### PR TITLE
Carshop project

### PR DESCRIPTION
**Pull request by anvaraxadjonov1802**
Programming task 01

### CODE REVIEW

File: car/main.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+            return True
+        elif answer == "No" or answer == "NO" or answer == "no":
+            return False
+        else: 
+            self.result_print()
+    
+
+
+if __name__ == "__main__":
+    print("""
```
Comment (Shaxzodbek16) [inline car/main.py:205]: you should add main() function to run this.

Comment (anvaraxadjonov1802) [reply car/main.py]: Is this done wrong?

**Review by anvaraxadjonov1802 — COMMENTED** [review]
Asosiy() funksiyasini qo'shgandan so'ng vazifa o'zgartirildi

File: car/main.py (hunk 1 above)
Comment (Shaxzodbek16) [reply car/main.py]: Yes it is, add main entry point to your script.

Comment (Shaxzodbek16) [reply car/main.py]: Yeah, I'll wait for you

**Comment by anvaraxadjonov1802** [comment]
Is it right ?

**Comment by Shaxzodbek16** [comment]
It was accepted

**Comment by Shaxzodbek16** [comment]
and you rename the car folder to your name

**Pull request merged**